telemetry/definitions/commonDefinitions.json

```diff
@@ -3645,6 +3645,48 @@
                 }
             ]
         },
+        {
+            "name": "cloudwatchlogs_startLiveTail",
+            "description": "When user starts a new LiveTail command",
+            "metadata": [
+                {
+                    "type": "filterType",
+                    "required": false
+                },
+                {
+                    "type": "hasTextFilter",
+                    "required": false
+                },
+                {
+                    "type": "result"
+                },
+                {
+                    "type": "sessionAlreadyStarted",
+                    "required": true
+                },
+                {
+                    "type": "source",
+                    "required": true
+                }
+            ]
+        },
+        {
+            "name": "cloudwatchlogs_stopLiveTail",
+            "description": "When user stops a liveTailSession",
+            "metadata": [
+                {
+                    "type": "duration",
+                    "required": true
```

Comment:
duration is a standard field. https://github.com/aws/aws-toolkit-common/blob/main/telemetry/telemetryformat.md#global-arguments

was there a client problem that prompted you to add this field here?

Reply:
No client problem. In the PR where these were initially added it was commented on to add these fields. If they are not explicitly needed then I will remove them